src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -26,6 +26,18 @@ public interface JsonApiMetricsConfig {
   @WithDefault("sort.type")
   String sortType();
 
+  @NotBlank
+  @WithDefault("json.shredding.type")
+  String jsonShreddingTypeTag();
+
+  @NotBlank
+  @WithDefault("json.string.length")
+  String jsonStringLength();
```

Comment:
I think we need to figure out better names: we are not exposing String lengths: we do measure String length but report size of json document, for serialization.

So let's figure out good naming after resolving other issues